jolly-jellyfish/src/page_maker/templates/registration/login.html

```diff
@@ -0,0 +1,23 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+{% if form.errors %}
```

Comment:
```suggestion
{% block content %}
<h2>Log in</h2>
{% if form.errors %}
```
Still missing a header.